src/nvhttp.cpp

```diff
@@ -278,17 +282,17 @@ namespace nvhttp {
       if (x == 2) launch_session.fps = atoi(segment.c_str());
       x++;
     }
-    launch_session.unique_id = (get_arg(args, "uniqueid"));
-    launch_session.uuid = (get_arg(args, "uuid"));
-    launch_session.appid = util::from_view(get_arg(args, "appid"));
-    launch_session.enable_sops = util::from_view(get_arg(args, "sops"));
-    launch_session.surround_info = util::from_view(get_arg(args, "surroundAudioInfo"));
-    launch_session.gcmap = util::from_view(get_arg(args, "gcmap"));
+    launch_session.unique_id = (get_arg(args, "uniqueid", "unknown"));
+    launch_session.uuid = (get_arg(args, "uuid", "unknown"));
+    launch_session.appid = util::from_view(get_arg(args, "appid", "unknown"));
+    launch_session.enable_sops = util::from_view(get_arg(args, "sops", "0"));
+    launch_session.surround_info = util::from_view(get_arg(args, "surroundAudioInfo", "0"));
```

Comment:
`surroundAudioInfo`  default value should be `196610` which corresponds to stereo channel count and channel mask.